src/index.html

```diff
@@ -1,14 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__container">
+        <a href="#" class="header__logo">
+          <img src="/images/logo.png" alt="logo icon">
+        </a>
+        <nav class="header__menu">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a class="menu__link is-active" href="#">Apple</a>
+            </li>
```

Comment:
якщо по бему то menu__list це неправильний клас, немає окремого блоку menu щоб від нього відштовхуватись, можна написати так
```suggestion
        <nav class="header__menu nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a class="nav__link is-active" href="#">Apple</a>
            </li>
```

Reply:
Понял, подправлю (: